lib/broc-component-css-preprocessor.js

```diff
@@ -92,7 +93,21 @@ BrocComponentCssPreprocessor.prototype.write = function (readTree, destDir) {
         var podGuid = podPath.join('--') + '-' + guid();
         var fileContents = fs.readFileSync(path.join(srcDir, filepath)).toString();
 
-        buffer.push(processors[pod.extension](fileContents, podGuid));
+        if(pod.extension === 'scss') {
```

Comment:
I think you should switch this to `if (pod.extension !== 'css') {` since we'll use this for other preprocessors as well.